tests/conftest.py

```diff
@@ -42,6 +42,16 @@
 from tests.setup_nodes import setup_farmer_multi_harvester
 
 
+async def wallet_is_synced(wallet_node: WalletNode, full_node_api) -> bool:
+    wallet_height = await wallet_node.wallet_state_manager.blockchain.get_finished_sync_up_to()
+    full_node_height = full_node_api.full_node.blockchain.get_peak_height()
+    return wallet_height == full_node_height
+
+
+async def wallets_are_synced(wns: List[WalletNode], full_node_api) -> bool:
+    return all([await wallet_is_synced(wn, full_node_api) for wn in wns])
+
+
```

Comment:
I think this belongs in a test util file.  In my experience `conftest.py` files are just there for pytest fixtures. So far that applies here as well.

Reply:
I think this should be changed since it is structural regression.

Reply:
yep makes sense